config/settings/base.py

```diff
@@ -328,7 +328,7 @@
     },
 }
 
-DJANGO_DATADOG_LOGGER_EXTRA_INCLUDE = r"django_datadog_logger\.middleware\.request_log"
+DJANGO_DATADOG_LOGGER_EXTRA_INCLUDE = r"^(django_datadog_logger\.middleware\.request_log|itou)(|\..+)$"
```

Comment:
Je n’ai pas trop cherché à le tester, mais utiliser `re.compile` devrait accélérer le traitement, et nous permet d’utiliser le flag verbose pour clarifier la regexp.
```suggestion
DJANGO_DATADOG_LOGGER_EXTRA_INCLUDE = re.compile(
    r"""
    ^(
        django_datadog_logger\.middleware\.request_log  # Built-in datadog logger.
        |itou(\..+)?  # Itou root logger and children.
    )$""",
    re.VERBOSE,
)
```